pkg/utils/options/options.go

```diff
@@ -27,6 +27,7 @@ func MustParse() Options {
 	opts := Options{}
 	flag.StringVar(&opts.ClusterName, "cluster-name", env.WithDefaultString("CLUSTER_NAME", ""), "The kubernetes cluster name for resource discovery")
 	flag.StringVar(&opts.ClusterEndpoint, "cluster-endpoint", env.WithDefaultString("CLUSTER_ENDPOINT", ""), "The external kubernetes cluster endpoint for new nodes to connect with")
+	flag.StringVar(&opts.DefaultInstanceProfile, "default-instance-profile", env.WithDefaultString("DEFAULT_INSTANCE_PROFILE", ""), "The default instance profile to use when provisioning nodes")
```

Comment:
thoughts on prepending this w/ `AWS`? I would think when we do have cloud providers register flags, we'll want to namespace them within their cloud provider, so prepending with the cloud provider now may save us a backwards incompatible change later :)

Reply:
I'm good with this idea and can make the adjustment